core/src/main/java/tech/pegasys/eth2signer/core/signing/filecoin/FilecoinAddress.java

```diff
@@ -87,18 +88,19 @@ public static FilecoinAddress decode(final String address) {
       final Bytes payload = putUVariant(new BigInteger(rawPayload));
       return new FilecoinAddress(protocol, payload);
     } else {
-      if (!base32.isInAlphabet(rawPayload)) {
+      // TODO(tmm): The toUpper shouldn't be required.
```

Comment:
Is this TODO coming in another PR?

Reply:
hope so? But do not want to investigate it here.